### ISSUE TITLE
[REQUEST] Use Gyro as Gyro in others Apps

### ISSUE THREAD

**Issue by SiTWulf** [Ryochan7/DS4Windows#98]
Hi.
Sorry about my english, is not my native language.

Please include the Gyro patch for others apps like this patched version:
https://github.com/rajkosto/DS4Windows

More info in here:
https://sshnuke.net/cemuhook/padudpserver.html

Is perfect for emulators like Cemu (WiiU) and others.

**Comment by Ryochan7** [comment]
rajkosto sent me a pull request with those changes a little while ago. However, I opted to reject the pull request. The main reasons that the pull request was rejected were due to all gyro functionality being broken with the changes and overall application performance degrading. A significant amount of input lag was introduced with the changes. If the base use case of turning a DS4 into an Xbox 360 controller is sacrificed to try to add functionality then I consider it a failure. There have been times when I had to attempt to tweak changes for new functionality in order to keep XInput performance working well; one recent example would be interpreting the timestamp data from the controller. Luckily, many attempts unintentionally led to improving performance.

Some ideas from the pull request have been implemented somewhat and a decent amount of refactoring has been done regarding how gyro data is interpreted. One major piece of functionality that is missing is calibration. My controller seems to work fine without having to use any calibration routine so it has not been a priority. This seems like a niche use case that I might have to try out in Cemu sometime; I have never used that emulator. I will keep watching for any progress made but this functionality will not be added into this repository as it is now.

https://github.com/Ryochan7/DS4Windows/pull/86
